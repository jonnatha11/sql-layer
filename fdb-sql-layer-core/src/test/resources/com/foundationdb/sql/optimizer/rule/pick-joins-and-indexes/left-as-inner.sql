SELECT orders.cid FROM orders INNER JOIN items ON orders.oid = items.oid
 WHERE sku = '1234' AND order_date = '2010-01-01'