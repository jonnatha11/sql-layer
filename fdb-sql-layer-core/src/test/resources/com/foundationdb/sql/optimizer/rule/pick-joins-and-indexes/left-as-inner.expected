SelectQuery@11575ea3
  ResultSet@7f062839[cid]
    Project@6d3af755[orders.cid]
      Select@3a442579[]
        TableGroupJoinTree@4ff848d1(TableGroup@42e97022(customers), orders INNER items - SingleIndexScan@6bc0ea7f(Index(test.customers.sku_and_date[IndexColumn(sku), IndexColumn(order_date)]), covering/NONE, ='1234', =DATE '2010-01-01', rows = 1, cost = 6.35760))
          TableSource@1e079362(orders - TableGroup@42e97022(customers))
            TableSource@35d1655(items - TableGroupJoin@3b8ada76(Join(test/orders/oid/test/test.items/oid: test.items -> test.orders, group(test.customers))))
